### PR TITLE
chore(example): update example to be async / await

### PR DESCRIPTION
**Pull request by cnishina**

### CODE REVIEW

File: example/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (juliemr) [inline example/.gitignore]: Why this change?

File: example/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "author": "",
+  "license": "MIT",
+  "scripts": {
+    "debug": "node --inspect-brk ./node_modules/.bin/protractor conf.js",
+    "pretest": "npm run webdriver-update",
+    "test": "protractor conf.js",
+    "webdriver-update": "webdriver-manager update --standalone=false --gecko=false"
+  },
+  "dependencies": {
+    "@types/jasmine": "^3.3.12",
```
Comment (juliemr) [inline example/package.json]: Why add types if it's not TS yet?

Comment (cnishina) [reply example/package.json]: 🤦‍♂️

File: example/.gitignore (hunk 1 above)
Comment (cnishina) [reply example/.gitignore]: Not required. I added a package.json to the example/ directory.

Comment (cnishina) [reply example/.gitignore]: Removed.

File: example/package.json (hunk 2 above)
Comment (cnishina) [reply example/package.json]: Removed.
